app/demo-loop/LDemoIO.cc

```diff
@@ -68,8 +90,8 @@ TransporterInput load_input(const LDemoArgs& args)
     CELER_LOG(status) << "Loading input files";
     TransporterInput result;
 
-    // Load data from ROOT file
-    const auto data = RootImporter(args.physics_filename.c_str())();
+    // Load rootinp from ROOT file
+    auto rootinp = RootImporter(args.physics_filename.c_str())();
```

Comment:
```
< auto rootinp = RootImporter(args.physics_filename.c_str())();
> auto rootimp = RootImporter(args.physics_filename.c_str())();
```
and/or what is the semantinc of the 'n' :) ?

Reply:
Import the input!